deltachat-ios/Controller/Settings/Pages/README.md

```diff
@@ -0,0 +1,10 @@
+#  Settings Pages
+
+This group is for settings pages that combine multiple settings on one page.
```

Comment:
by this definition, `AutodeletionOverviewViewController` is a page and not a single setting, or?

tbh, i am not convinced by trying to sort all setting view controllers into "single" and "pages" as artificial categories, it adds bureaucracy and discussions, for a handful of files. 

so, +1 for streamlining the names, but a flat structure seems "good enough" here.

Reply:
I think it's faster to find, if you search for a page, but yeah `AutodeletionOverviewViewController` should be moved.

Reply:
it is only faster to find if you know beforehand, if sth. is categorized as page or a single option. and you have to understand these categories. also, things may move between them ... eg. the VideoChatViewController you just updated may easily become a "page". we can think over it if we have several dozens of files and controllers, let us just keep things simple here :)

I understand the goal you have here, but i don't think i agree that it hits the correct set of tradeoffs.